<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_DatabaseConnectionWidget</class>
 <widget class="QWidget" name="w_DatabaseConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_driverName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Sterownik</string>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="f_driverName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_hostName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>35</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa hosta</string>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="l_Port">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="l_dbName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>105</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa Bazy danych</string>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="l_userName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa użytkownika</string>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="l_password">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>175</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasło</string>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_hostName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>35</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_port">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_dbName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>106</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_userName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_password">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>175</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="testConnectionButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
